### ISSUE TITLE
Sometimes, when two recordings start at the same time, I get "tuner is in use by another process"

### ISSUE THREAD

**Issue by scumola** [tgorgdotcom/locast2plex#207]
![image](https://user-images.githubusercontent.com/386016/106496382-0fdf1d00-647a-11eb-8359-14299b1f8c31.png)
![image](https://user-images.githubusercontent.com/386016/106496625-5765a900-647a-11eb-8447-82764ca79e0b.png)

Both recordings started at 10am, but the second one failed to start due to the error, "tuner is in use by another process".  Not sure if this is plex or locast2plex.  Container is running on docker swarm under ubuntu 20.04 and is otherwise running well.

**Comment by CTJohnK** [comment]
What value are you using for "tuner_count"?  Also, how long before and/or after the program did you tell Plex to record?

**Comment by scumola** [comment]
I'm using 4 tuners as the tuner count.  When the problem shows up it happens when shows are scheduled to start at the exact same time with no 'early start' time in the schedule.  My work-around now is to schedule some shows to start 1-minute early if I want to avoid the "tuner is in use" error.

**Comment by CTJohnK** [comment]
And you don't have the recording time extended either, correct?On Feb 1, 2021 3:18 PM, Steve Webb <notifications@github.com> wrote:
I'm using 4 tuners as the tuner count.  When the problem shows up it happens when shows are scheduled to start at the exact same time with no 'early start' time in the schedule.  My work-around now is to schedule some shows to start 1-minute early if I want to avoid the "tuner is in use" error.

**Comment by scumola** [comment]
Correct.  No recording times have been extended or started early in the example shown.  The scheduled shows don't exceed the 4-tuner limit even when taking into account any early start or late finish extensions.  Very seldomly > 2 recordings at once.  In the example shown, there were only two recordings scheduled to start at 10am with one (The View) ending at 10am.  None of them had early starts or late finishes, so even if "The View" stopped late, that would only be 3 tuners in use for all of the recordings.

**Comment by CTJohnK** [comment]
Strange, I record Doctor Who and Finding your roots at the same time at 8pm eastern with out issue.On Feb 1, 2021 3:35 PM, Steve Webb <notifications@github.com> wrote:
Correct.  No recording times have been extended or started early in the example shown.  The scheduled shows don't exceed the 4-tuner limit even when taking into account any early start or late finish extensions.  Very seldomly > 2 recordings at once.  In the example shown, there were only two recordings scheduled to start at 10am with one (The View) ending at 10am.  None of them had early starts or late finishes, so even if "The View" stopped late, that would only be 3 tuners in use for all of the recordings.

**Comment by scumola** [comment]
It doesn't happen all of the time though.  Intermittent.
